extensions/replication/queueProcessor/QueueProcessor.js

```diff
@@ -1019,6 +1020,18 @@ class QueueProcessor extends EventEmitter {
      */
     static async handleMetrics(res, log) {
         log.debug('metrics requested');
+
+        if (this.repConfig.queueProcessor.logConsumerMetricsIntervalS) {
+            // consumer stats lag is on a different update cycle so we need to
+            // update the metrics when requested
+            const lagStats = this._consumer.consumerStats.lag;
+            Object.keys(lagStats).forEach(partition => {
+                metricsHandler.lag({
```

Comment:
The lag metric needs to be added
https://github.com/scality/backbeat/blob/57fc2357d1bc9fe477b0f38f012f2e3633f68f70/extensions/replication/queueProcessor/QueueProcessor.js#L62-L66